### PR TITLE
monitor parameter

### PR DESCRIPTION
**Pull request by izhuxiaoqing**

### CODE REVIEW

File: docs-2.0/nebula-dashboard/6.monitor-parameter.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+### CPU
+
+|Parameter|Description|
+|:---|:---|
+|`cpu_utilization`| The percentage of used CPU. |
+|`cpu_idle`| The percentage of idled CPU. |
+|`cpu_wait`| The percentage of CPU waiting for IO operations. |
+|`cpu_user`| The percentage of CPU used by users (not the graph space of Nebula Graph). |
+|`cpu_system`| The percentage of CPU used by the system (not the kernel space of Nebula Graph). |
```
Comment (randomJoe211) [inline docs-2.0/nebula-dashboard/6.monitor-parameter.md:19]:
```suggestion
|`cpu_user`| The percentage of CPU used by users |
|`cpu_system`| The percentage of CPU used by the system |
```

And remove all periods at the end of phrases.

File: docs-2.0/nebula-dashboard/6.monitor-parameter.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+### Memory
+
+|Parameter| Description|
+|:---|:---|
+|`memory_utilization`| The percentage of used memory. |
+|`memory_used`| The memory that is used. |
+|`memory_actual_used`| The memory that is in actual use. |
```
Comment (randomJoe211) [inline docs-2.0/nebula-dashboard/6.monitor-parameter.md:27]: Clarify these two.

File: docs-2.0/nebula-dashboard/6.monitor-parameter.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+|`memory_free`| The memory that is idled. |
```
Comment (randomJoe211) [inline docs-2.0/nebula-dashboard/6.monitor-parameter.md:28]: Add the size of to this part? Like The size of free memory.

File: docs-2.0/nebula-dashboard/6.monitor-parameter.md [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+|`cpu_wait`| The percentage of CPU waiting for IO operations. |
+|`cpu_user`| The percentage of CPU used by users (not the graph space of Nebula Graph). |
+|`cpu_system`| The percentage of CPU used by the system (not the kernel space of Nebula Graph). |
+
+### Memory
+
+|Parameter| Description|
+|:---|:---|
+|`memory_utilization`| The percentage of used memory. |
+|`memory_used`| The memory that is used. |
```
Comment (randomJoe211) [inline docs-2.0/nebula-dashboard/6.monitor-parameter.md:26]: The size of used memory or The memory space used?

File: docs-2.0/nebula-dashboard/6.monitor-parameter.md [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+|:---|:---|
+|`load_1m`| The average load of the system in the last 1 minute. |
+|`load_5m`| The average load of the system in the last 5 minutes. |
+|`load_15m`| The average load of the system in the last 15 minutes. |
+
+### Disk
+
+|Parameter| Description|
+|:---|:---|
+|`disk_used`| The disk that is used. |
```
Comment (randomJoe211) [inline docs-2.0/nebula-dashboard/6.monitor-parameter.md:42]: The disk space?

File: docs-2.0/nebula-dashboard/6.monitor-parameter.md [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+|`disk_free`| The disk that is idled. |
+|`disk_readbytes`| The number of bytes that the disk reads every second. |
+|`disk_writebytes`| The number of bytes that the disk writes every second. |
```
Comment (randomJoe211) [inline docs-2.0/nebula-dashboard/6.monitor-parameter.md:45]: The subject should be the system.

**Review by whitewum — APPROVED** [review]

File: docs-2.0/nebula-dashboard/6.monitor-parameter.md [hunk 7]
```diff
@@ -0,0 +1,90 @@
...
+### Network
+
+|Parameter| Description|
+|:---|:---|
+|`network_in_rate`| The number of bytes that the network receives every second. |
+|`network_out_rate`| The number of bytes that the network sends out every second. |
+|`network_in_errs`| The number of wrong bytes that the network receives every second. |
+|`network_out_errs`| The number of wrong bytes that the network sends out every second. |
+|`network_in_packets`| The number of data packages that the network receives every second. |
+|`network_out_packets`| The number of data packages that the network sends out every second. |
```
Comment (randomJoe211) [inline docs-2.0/nebula-dashboard/6.monitor-parameter.md:59]: Network card?

File: docs-2.0/nebula-dashboard/6.monitor-parameter.md [hunk 8]
```diff
@@ -0,0 +1,90 @@
...
+
+|Parameter| Description|
+|:---|:---|
+|`add_edges_atomic_latency_us`| Add a single latency of edges. |
+|`add_edges_latency_us`| Add the average latency of edges. |
+|`add_vertices_latency_us`| Add the average latency of vertices. |
+|`delete_edges_latency_us`| Delete the average latency of edges. |
+|`delete_vertices_latency_us`| Delete the average latency to vertices. |
+|`forward_tranx_latency_us`| Transmit the average latency. |
+|`get_neighbors_latency_us`| Query the average latency of neighbors. |
```
Comment (randomJoe211) [inline docs-2.0/nebula-dashboard/6.monitor-parameter.md:90]: Double-check this part.

**Review by randomJoe211 — CHANGES_REQUESTED** [review]

**Review by randomJoe211 — APPROVED** [review]

File: docs-2.0/nebula-dashboard/6.monitor-parameter.md [hunk 9]
```diff
@@ -0,0 +1,89 @@
...
+### Network
+
+|Parameter| Description|
+|:---|:---|
+|`network_in_rate`| The number of bytes that the network card receives every second. |
+|`network_out_rate`| The number of bytes that the network card sends out every second. |
+|`network_in_errs`| The number of wrong bytes that the network card receives every second. |
+|`network_out_errs`| The number of wrong bytes that the network card sends out every second. |
+|`network_in_packets`| The number of data packages that the network card receives every second. |
+|`network_out_packets`| The number of data packages that the network card sends out every second. |
```
Comment (Kun8018) [inline docs-2.0/nebula-dashboard/6.monitor-parameter.md:59]: replace every with per is better?~

**Review by Kun8018 — APPROVED** [review]

File: docs-2.0/nebula-dashboard/6.monitor-parameter.md [hunk 10]
```diff
@@ -0,0 +1,89 @@
...
+|`cpu_system`| The percentage of CPU used by the system. |
+
+### Memory
+
+|Parameter| Description|
+|:---|:---|
+|`memory_utilization`| The percentage of used memory. |
+|`memory_used`| The memory space used (including caches). |
+|`memory_actual_used`| The memory space used (not including caches). |
+|`memory_free`| The memory space idled. |
```
Comment (Aiee) [inline docs-2.0/nebula-dashboard/6.monitor-parameter.md:28]: The memory space that is available.

File: docs-2.0/nebula-dashboard/6.monitor-parameter.md [hunk 11]
```diff
@@ -0,0 +1,89 @@
...
+|`load_1m`| The average load of the system in the last 1 minute. |
+|`load_5m`| The average load of the system in the last 5 minutes. |
+|`load_15m`| The average load of the system in the last 15 minutes. |
+
+### Disk
+
+|Parameter| Description|
+|:---|:---|
+|`disk_used`| The disk space used. |
+|`disk_free`| The disk space idled. |
```
Comment (Aiee) [inline docs-2.0/nebula-dashboard/6.monitor-parameter.md:43]: ditto.

**Review by whitewum — APPROVED** [review]

**Review by whitewum — APPROVED** [review]

**Review by whitewum — APPROVED** [review]

**Review by randomJoe211 — APPROVED** [review]

**Pull request merged**